doc/api/events.md

```diff
@@ -2334,6 +2334,22 @@ equivalent `EventEmitter` API. The only difference between `addListener()` and
 `addEventListener()` is that `addListener()` will return a reference to the
 `EventTarget`.
 
+#### `nodeEventTarget.emit(type, arg)`
+
+<!-- YAML
+added: v15.2.0
+-->
+
+* `type` {string}
+
+* `arg` {any}
+
+* Returns: {boolean} `true` if event listeners registered for the `type` exist,
+  otherwise `false`.
```

Comment:
```suggestion
* `type` {string}
* `arg` {any}
* Returns: {boolean} `true` if event listeners registered for the `type` exist,
  otherwise `false`.
```

Reply:
Applied your suggestion. Thank you for suggestion :)